src/contracts/pods/EigenPodManager.sol

```diff
@@ -266,17 +266,18 @@ contract EigenPodManager is
 
     /**
      * @notice Called by a staker who owns an EigenPod to enter the "undelegation limbo" mode.
+     * @param undelegateIfPossible If this param is marked as 'true', then this function will also inform the DelegationManager of the caller's desire to undelegate
      * @dev Undelegation limbo is a mode which a staker can enter into, in which they remove their virtual "beacon chain ETH shares" from EigenLayer's delegation
      * system but do not necessarily withdraw the associated ETH from EigenLayer itself. This mode allows users who have restaked native ETH a route via
      * which they can undelegate from an operator without needing to exit any of their validators from the Consensus Layer.
      */
-    function enterUndelegationLimbo()
+    function enterUndelegationLimbo(bool undelegateIfPossible)
```

Comment:
you only go into undelegation limbo if you want to undelegate? :)

Reply:
hmm yeah good point, this probably just shouldn't have an input param, will make that change

Reply:
removed the input in this commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/176/commits/3c4c5e6bed5f94e0ae329046b041373aa250a67f